src/connectedk8s/azext_connectedk8s/_params.py

```diff
@@ -27,6 +27,7 @@ def load_arguments(self, _):
         c.argument('proxy_cert', options_list=['--proxy-cert'], type=file_type, completer=FilesCompleter(), help='Path to the certificate file for proxy')
         c.argument('distribution', options_list=['--distribution'], help='The Kubernetes distribution which will be running on this connected cluster.', arg_type=get_enum_type(Distribution_Enum_Values))
         c.argument('infrastructure', options_list=['--infrastructure'], help='The infrastructure on which the Kubernetes cluster represented by this connected cluster will be running on.', arg_type=get_enum_type(Infrastructure_Enum_Values))
+        c.argument('auto_upgrade', options_list=['--auto-upgrade'], help='Flag to enable/disable auto upgrade of arc agents. By default, auto upgrade of agents is enabled.', arg_type=get_enum_type(["true", "false"]))
```

Comment:
Let's keep the flag as --disable-auto-upgrade since by default its true. So the  customer will just specify the flag --disable-auto-update when he dosent wish to enable it. For enabling auto update, this flag is not reqd(since that is the default behavior). For reference you can see this flag: https://github.com/Azure/azure-cli/blob/785d8f4fcb923a7a9215d7217b77d819191ecd71/src/azure-cli/azure/cli/command_modules/vm/_params.py#L341

Reply:
This i had thought earlier, but according to the spec and toggling/switching experience for customer, i made a call for this. Will look into this "three_state_flag" thing.

Reply:
In connect command, we can keep --disable-auto-upgrade